CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Minimum Node version is now 18.0.0 ([#620](https://github.com/STORIS/mvom/pull/620))
```

Comment:
You lost the section headings for the change type in your last commit. Those should still be here.